intake/cli/client/subcommands/__init__.py

```diff
@@ -0,0 +1,19 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2018, Anaconda, Inc. and Intake contributors
+# All rights reserved.
+#
+# The full license is in the LICENSE file, distributed with this software.
+#-----------------------------------------------------------------------------
+
+from .cache import Cache
+from .config import Config
+from .describe import Describe
+from .discover import Discover
+from .example import Example
+from .exists import Exists
+from .get import Get
+from .info import Info
+from .list import List
+from .precache import Precache
+
+all  = (Cache, Config, Describe, Discover, Example, Exists, Get, Info, List, Precache)
```

Comment:
This can be automated if you like, e.g. https://github.com/bokeh/bokeh/blob/master/bokeh/command/subcommands/__init__.py#L49